src/commands/assign.js

```diff
@@ -97,9 +97,37 @@ function setPrompt() {
   }
 
   // Assigned info.
+  //
+  // Shows the number of projects that are currently assigned
   console.log(chalk.green(`Currently assigned: ${chalk.white(assignedCount)}`));
+
+  // Shows assigned projects in a table
+  const submissionDetails = new Table({
+    head: [
+      {hAlign: 'center', content: 'sub'},
+      {hAlign: 'center', content: 'id'},
+      {hAlign: 'left', content: 'name'},
+      {hAlign: 'center', content: 'time'}],
+    colWidths: [10, 7, 40, 7],
+  });
+
+  assigned
+    .forEach((submission) => {
+      submissionDetails.push([
+        {hAlign: 'center', content: submission.submission_id},
+        {hAlign: 'center', content: submission.project_id},
+        {hAlign: 'left', content: certs[submission.project_id].name},
+        {hAlign: 'center', content: submission.timeToFinish},
```

Comment:
I think it is calculated every time the `checkAssigned`function is run, which is at every 5 minutes, I think.